app/leave-app/backend/modules/calendar_events/client.bal

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) 2024 WSO2 LLC. (http://www.wso2.org).
+//
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+import ballerina/http;
+
+configurable string eventBaseUrl = ?;
+configurable ChoreoApp choreoAppConfig = ?;
+
+public final http:Client eventClient = check new (eventBaseUrl, {
+    auth: {
+        ...choreoAppConfig
+    },
+    httpVersion: http:HTTP_1_1,
+    http1Settings: {keepAlive: http:KEEPALIVE_NEVER},
```

Comment:
What is the functionality obtained by these configurations in this service context?

Reply:
This was added as a precaution due to recently occurred issue(`Subj:Experiencing intermittent timeouts in most Internal Apps and Integrations running on Choreo`) apart from setting `minEvictableIdleTime`.